xarray/tests/test_parallelcompat.py

```diff
@@ -218,3 +220,13 @@ def test_raise_on_mixed_array_types(self, register_dummy_chunkmanager) -> None:
 
         with pytest.raises(TypeError, match="received multiple types"):
             get_chunked_array_type(*[dask_arr, dummy_arr])
+
+
+def test_bogus_entrypoint():
```

Comment:
```suggestion
def test_bogus_entrypoint() -> None:
```
Let's see if this fixes the mypy error...